themes/bootstrap5/templates/RecordDriver/DefaultRecord/result-list.phtml

```diff
@@ -198,15 +198,15 @@
       <a class="hidden full-record-link icon-link" href="<?=$recordLinker->getUrl($this->driver)?>">
         <?=$this->icon('link', 'icon-link__icon') ?>
         <span class="icon-link__label"><?=$this->transEsc('standalone_record_link')?></span>
-      </a><br>
+      </a>
 
       <?php if ($this->userlist()->getMode() !== 'disabled'): ?>
         <?php if ($this->permission()->allowDisplay('feature.Favorites')): ?>
           <?php /* Add to favorites */ ?>
           <a href="<?=$this->escapeHtmlAttr($recordLinker->getActionUrl($this->driver, 'Save'))?>" data-lightbox class="save-record result-link icon-link" data-id="<?=$this->escapeHtmlAttr($this->driver->getUniqueId()) ?>">
             <?=$this->icon('user-favorites', 'icon-link__icon') ?>
             <span class="result-link-label icon-link__label"><?=$this->transEsc('Add to favorites')?></span>
-          </a><br>
+          </a>
```

Comment:
There are several variant result-list.phtml files (e.g. for EDS and SolrOverdrive) that I suspect may contain the same or similar logic. Did you look at any of those? We should probably check and standardize everything before we merge this. Let me know if you need help testing any obscure corners of the code!